Gemfile

```diff
@@ -26,7 +26,7 @@ gem 'azure_mgmt_dns',                       path: 'management/azure_mgmt_dns'
 gem 'azure_mgmt_event_hub',                 path: 'management/azure_mgmt_event_hub'
 gem 'azure_mgmt_features',                  path: 'management/azure_mgmt_features'
 gem 'azure_mgmt_graph',                     path: 'management/azure_mgmt_graph'
-gem 'azure_mgmt_insights',                  path: 'management/azure_mgmt_insights'
+gem 'azure_mgmt_monitor',                   path: 'management/azure_mgmt_monitor'
```

Comment:
The insights service has been renamed to monitor. Discussed with @salameer and decided to change it here also. Yes. If the customers want to move to this service, this will be a breaking change.